secure_message/logger_config.py

```diff
@@ -26,7 +26,7 @@ def logger_initial_config(service_name=None,
     except ValueError:
         indent = None
 
-    def add_service(logger, method_name, event_dict):
+    def add_service(_1, _2, event_dict):
```

Comment:
Would it better not to have `_1, _2` and have `_, _` ?

Reply:
No as that causes an error , the _X syntax allows multiple unused variables. The better answer would be to convert to a named tuple

Reply:
Have changed to a named tuple approach , makes it cleaner .